app/src/main/res/layout/topic_lessons_story_summary.xml

```diff
@@ -156,7 +156,7 @@
           android:id="@+id/chapter_recycler_view"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginStart="56dp"
+          android:layout_marginStart="5dp"
```

Comment:
We keep margin and padding normally in multiple of 4. Therefore 5 is incorrect. At the same time, you can remove this marginStart completely and that would look decent enough for now.

Reply:
@rt4914 I updated the file with the latest commit, I made a new PR in which I did not include those .idea files , please check if it is is reflecting at your end.

Reply:
@rt4914 Made another commit, this time did not include those .idea files and removed marginStart completely.